docs/semgrep-sc/sc-getting-started.md

```diff
@@ -28,12 +29,14 @@ Object.entries(frontMatter).filter(
 }
 </ul>
 
-# Scanning open source dependencies 
+# Getting started with Semgrep Supply Chain
 
-Semgrep Supply chain scans can be triggered from the Semgrep App interface.
+Semgrep Supply Chain detects recently discovered [security vulnerabilities](https://nvd.nist.gov/vuln/full-listing) in your codebase's open source dependencies, prioritizing findings through [reachability](/docs/semgrep-sc/sc-glossary#reachability) analysis. 
+
+This document walks you through the process of setting up open source dependency scanning with Semgrep Supply Chain.
 
 :::note
-Semgrep Supply Chain **supports monorepositories (monorepos)** by treating each subdirectory as its own repository. Findings are grouped under these repositories based on the lockfile or manifest file present in the subdirectory.
+Semgrep Supply Chain **supports monorepositories (monorepos)** by treating each subdirectory as its own repository. Findings are grouped under these repositories based on the [lockfile](/docs/semgrep-sc/sc-glossary/#lockfile) or manifest file present in the subdirectory.
```

Comment:
Just a small note - for absolute internal links (which work the best when we are changing the structure of the docs, so a good choice), do not include the `/docs/` in the beginning. The root for links in `/` is considered as `/docs/` already. Otherwise, this causes some issues which you can see if you run `yarn build && yarn serve`. 🙂

```suggestion
Semgrep Supply Chain **supports monorepositories (monorepos)** by treating each subdirectory as its own repository. Findings are grouped under these repositories based on the [lockfile](/semgrep-sc/sc-glossary/#lockfile) or manifest file present in the subdirectory.
```